VersionControlUI/src/DiffManager.cpp

```diff
@@ -147,6 +227,9 @@ void DiffManager::computeDiff(QString oldVersion, QString newVersion, QList<Chan
 		auto oldNode = const_cast<Model::Node*>(diffSetup.oldVersionManager_->nodeIdMap().node(id));
 		auto newNode = const_cast<Model::Node*>(diffSetup.newVersionManager_->nodeIdMap().node(id));
 
+		if (isNameChange(oldNode, newNode, diffSetup))
```

Comment:
This avoids having highlights for name changes, since they are summarized in a message overlay at the moment.